### PR TITLE
Ethnic group for 17.0.1.2.1

### PR DESCRIPTION
**Pull request by kneckinator**
## **Why is this change needed?**
Add to OpenSPP 17.0.1.2.1 Batanes release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ addon | summary
 [spp_entitlement_basket](spp_entitlement_basket/) | This module allows you to define baskets of goods and services that beneficiaries are entitled to receive, simplifying in-kind entitlement management within social protection programs.
 [spp_entitlement_cash](spp_entitlement_cash/) | Manage cash-based entitlements for beneficiaries within social protection programs, including defining calculation rules, automating disbursement, and tracking payments.
 [spp_entitlement_in_kind](spp_entitlement_in_kind/) | Manages the distribution of in-kind entitlements within social protection programs, handling inventory, service points, and beneficiary redemption.
+[spp_ethnic_group](spp_ethnic_group/) | Manages ethnic group
```
Comment (kneckinator) [inline README.md:64]: @jeremi @dasunhegoda please elaborate on this description if needed, or let me know if we can release with the existing description I put there during the preparation of this PR

File: spp_ethnic_group/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+{
+    "name": "OpenSPP: Ethnic Group",
+    "category": "OpenSPP",
+    "version": "17.0.1.0.0",
+    "sequence": 1,
+    "author": "OpenSPP.org",
+    "website": "https://github.com/OpenSPP/openspp-modules",
+    "license": "LGPL-3",
+    "development_status": "Beta",
```
Comment (kneckinator) [inline spp_ethnic_group/__manifest__.py:12]: Is this development status correct?

File: spp_ethnic_group/static/description/index.html [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+h4 tt.docutils, h5 tt.docutils, h6 tt.docutils {
+  font-size: 100% }
+
+ul.auto-toc {
+  list-style-type: none }
+
+</style>
+</head>
+<body>
+<div class="document" id="openspp-entitlement-basket">
```
Comment (kneckinator) [inline spp_ethnic_group/static/description/index.html]: @reichie020212 please update the description and title

Comment (reichie020212) [reply spp_ethnic_group/static/description/index.html]: @kneckinator sure

Comment (reichie020212) [reply spp_ethnic_group/static/description/index.html]: @kneckinator done updating the index.html

File: spp_ethnic_group/__manifest__.py (hunk 2 above)
Comment (reichie020212) [reply spp_ethnic_group/__manifest__.py:12]: @kneckinator I changed it to Alpha since the index.html is in Alpha.

Comment (kneckinator) [reply spp_ethnic_group/__manifest__.py:12]: I think (@jeremi or @dasunhegoda to confirm) that the change should go in the other direction, changing it to beta in `index.html`. My reasoning is that all other OpenSPP modules are either in Beta or higher.

Comment (dasunhegoda) [reply spp_ethnic_group/__manifest__.py:12]: @kneckinator @reichie020212 : let's make it `beta` to be consistant.

File: README.md (hunk 1 above)
Comment (dasunhegoda) [reply README.md:64]: I asked @reichie020212, @Markpenalosa, and @emjay0921. Couldn't find a better desc. Maybe @gonzalesedwin1123 knows. If not, let's proceed with this.

File: spp_ethnic_group/__manifest__.py (hunk 2 above)
Comment (gonzalesedwin1123) [reply spp_ethnic_group/__manifest__.py:12]: I also agree that it should be beta. For future escalation of development_status, how do we determine what should be set in a module?

File: README.md (hunk 1 above)
Comment (kneckinator) [reply README.md:64]: Let's proceed "as-is".

File: spp_ethnic_group/__manifest__.py (hunk 2 above)
Comment (kneckinator) [reply spp_ethnic_group/__manifest__.py:12]: @reichie020212 FYI/A

Comment (kneckinator) [reply spp_ethnic_group/__manifest__.py:12]: @reichie020212 Could you please set the version to `beta` in the relevant files? Thanks.

Comment (reichie020212) [reply spp_ethnic_group/__manifest__.py:12]: Noted on this @kneckinator

Comment (reichie020212) [reply spp_ethnic_group/__manifest__.py:12]: @kneckinator done changing from Alpha to Beta

Comment (kneckinator) [reply spp_ethnic_group/__manifest__.py:12]: Awesome, thanks @reichie020212

**Pull request merged**